Slack.Automation/Promact.Core.Repository/TaskMailRepository/TaskMailRepository.cs

```diff
@@ -173,7 +173,7 @@ public TaskMailRepository(IRepository<TaskMail> taskMail, IStringConstantReposit
                                         // if previous question was hour of task and it was not null/wrong value then answer will ask next question
                                         var hour = Convert.ToDecimal(answer);
                                         // checking range of hours
-                                        if (hour > 0 && hour < 8)
+                                        if (hour > 0 && hour <= 8)
```

Comment:
TaskMailRepository has following issues.
1. Naming conventions
2. Wrong usage of framework API
3. Logical implementation is not optimal.

Please fix all of these issues before re-submitting this pull request.